managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+1. Click **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+2. Click a workflow. The default *Text* view of the workflow opens where you see the code content.
+
+   ![Text view](../images/embedworkflow_textview.png){: caption="Text view" caption-side="bottom"}
+
+   Alternatively, click the **Graph** tab to see a graph of the same data.
+
+   ![Graph view](../images/embedworkflow_graphView.png){: caption="Graph view" caption-side="bottom"}
+
+3. If your workflow has credential fields, then create a workflow credential for each of the fields. Then, map the workflow credential to the workflow.
+   <!-- Add substeps from development - A.G to provide.-->
+
+### Creating a Service Dialog using an Embedded Workflow
+
+With Embedded Workflows enabled, fields that are *Dynamic* offer the choice of *Embedded Workflows* in addition to the original *Embedded Automate*.
+
+Dialogs are the part of {{ site.data.product.title_short }} that you interface with when you order services.
+
+1. To open a service dialog, click **Automation** > **Embedded Automate** > **Customization**.
+2. From the *Service Dialog* section, review the list of created dialogs. Click one of the dialogs to view the details.
+3. Click **Edit the dialog**. You can edit the *Name*, *Provider*, and *Template*.
+
+   When you select to edit any of the dropdown settings, you can see a *dynamic* option. Toggle to **On** to make sure it is enabled.
+
+4. Click the **Options** tab. You can now select between *Embedded Automate* and *Embedded Workflows* in the Automation field.
+
+   ![Dialog Dynamic](../images/embedworkflow_servicedialogdynamic.png){: caption="Dialog Dynamic" caption-side="bottom"}
+
+   If you choose **Embedded Workflows**, you can select the workflow to run. Complete the fields, such as *Template* (using drag and drop functionality) and click **Save** when you complete the form. The workflow now runs and refreshes the fields with data from the workflow. In this way fields are chained together. When you *order* a service catalog, the workflow runs in the background to refresh the template.
+
+### Creating Service Catalog Items using an Embedded Workflow
+
+You can create a generic service catalog item that uses a created dynamic dialog and an embedded workflow. To use previously set up workflows and to view a service catalog item, you need to *order* the item. Complete the following steps to order a service catalog item that uses embedded workflows:
+
+1. Click **Services** > **Catalogs**.
+2. The *Service Catalogs* section opens and you see the existing Service Catalogs. If you want to order one, click it and then **Order**.
+
+   ![Service Dialog Dynamic](../images/embedworkflow_orderservicecatalog.png){: caption="Service Dialog Dynamic" caption-side="bottom"}
+
+   Ensure that the *VM Name*, *Provider* and *Template* fields are correct, and then click **Submit**.
```

Comment:
This is very specific to the example dynamic dialog that I've created have and is not generic.  We can probably leave this out or simply say "fill in the fields in your specified dialog then click submit" or something like that

Reply:
I have updated this sentence as requested. It is now:

> Complete the fields for your specified dialog. Then, click **Submit**.